app/frontend/errors/internalErrorReason.ts

```diff
@@ -0,0 +1,53 @@
+export enum InternalErrorReason {
+  SendAddressInvalidAddress,
```

Comment:
these enum values would be integers if I understand correctly, which is not consistent with how ExternalErrorReasons are declared and hence may cause confusion. Isn't this also related to @rizip1 's comment https://github.com/vacuumlabs/adalite/pull/911/files#r607883464 ?